templates/ingress.yaml

```diff
@@ -35,33 +35,36 @@ metadata:
     kubernetes.io/ingress.class: "gce"
     kubernetes.io/ingress.allow-http: "false"
     {{- end}}
-    {{- if .UseDNSAnnotationsOnIngress}}  
+    {{- if .UseDNSAnnotationsOnIngress}}
     estafette.io/cloudflare-dns: "true"
     estafette.io/cloudflare-proxy: "{{.UseCloudflareProxy}}"
     estafette.io/cloudflare-hostnames: "{{.HostsJoined}}"
     {{- end}}
 spec:
   tls:
-  - hosts:
-    {{- range .Hosts}}
-    - {{.}}
-    {{- end}}
-    {{- if .UseCertificateSecret }}
-    secretName: {{.CertificateSecretName}}
-    {{- else }}
-    secretName: {{.Name}}-letsencrypt-certificate
-    {{- end }}
+    - hosts:
+      {{- range .Hosts}}
+      - {{.}}
+      {{- end}}
+      {{- if .UseCertificateSecret }}
+      secretName: {{.CertificateSecretName}}
+      {{- else }}
+      secretName: {{.Name}}-letsencrypt-certificate
+      {{- end }}
   rules:
   {{- range .Hosts}}
   - host: {{.}}
     http:
       paths:
-      - path: {{$.IngressPath}}
-        backend:
-          serviceName: {{$.Name}}
-          {{- if $.HasOpenrestySidecar }}
-          servicePort: https
-          {{- else }}
-          servicePort: web
-          {{- end }}
+        - path: {{$.IngressPath}}
+          pathType: ImplementationSpecific
```

Comment:
I think it would be good to set this from a property and default to `Prefix` and for GCE ingress use `ImplementationSpecific`, since nginx ingress does already support `Prefix` (although it will work fine with `ImplementationSpecific` as well).

Reply:
this is resolved